cmd/pj-rehearse/main.go

```diff
@@ -165,6 +166,16 @@ func main() {
 	loggers := rehearse.Loggers{Job: logger, Debug: debugLogger.WithField(prowgithub.PrLogField, prNumber)}
 
 	executor := rehearse.NewExecutor(changedPresubmits, prNumber, o.candidatePath, jobSpec.Refs, o.dryRun, loggers, pjclient)
+
+	if executor.RehearsalJobCount > REHEARSAL_THRESHOLD {
```

Comment:
Are we sorting the jobs? For the same set of selected jobs for rehearsal, will we choose the same 15 every time?

Reply:
> +0, we only rehearse container tests for now, so the impact shouldn't be that large, but it's nice to have a safety net.

Yeah, it's not that we would attempt to build hundreds of clusters, but [Clayton's PR yesterday](https://github.com/openshift/release/pull/2834) would submit ~320 jobs at once.

Reply:
I'm not attempting to sample, I'm bailing out when too many jobs would be rehearsed.